bundle/Resources/views/themes/ngadmin/content_type/tab/view.html.twig

```diff
@@ -0,0 +1,106 @@
+{% import "@ibexadesign/content_type/macros.html.twig" as macros %}
+
+{% trans_default_domain 'content_type' %}
+{% set current_language = app.request.get('languageCode') ?: language_code %}
+
+{% set url = path('ibexa.search', { 'search[content_types][]': content_type.identifier }) %}
+{% set object_count %}
+    <a href="{{ url }}"> {{ count_by_content_type(content_type.identifier)|default(0) }} </a>
```

Comment:
I think default is not necessary here.

Reply:
fixed in 0adfb30070f0a463f45a64f531126c8e7f683d99